akka-docs/src/main/paradox/stream/stream-introduction.md

```diff
@@ -18,7 +18,7 @@ another. We have found it tedious and error-prone to implement all the proper
 measures in order to achieve stable streaming between actors, since in addition
 to sending and receiving we also need to take care to not overflow any buffers
 or mailboxes in the process. Another pitfall is that Actor messages can be lost
-and must be retransmitted in that case lest the stream have holes on the
+and must be retransmitted in that case let the stream have holes on the
```

Comment:
What about this ?

`Another pitfall is that Actor messages can be lost
and must be retransmitted in order to avoid the stream have holes on the
receiving side` 

I am not a native english speaker, but my sentence is always correct to me ;)

Reply:
"to avoid the stream have holes" does not quite sound right to me. Perhaps split the sentence: "Another pitfall is that Actor messages can be lost and must be retransmitted in that case. Failure to do so would lead to holes at the receiving side"?